README.md

```diff
@@ -50,89 +25,68 @@ Plugin 'rescript-lang/vim-rescript'
 
 " NeoBundle
 NeoBundle 'rescript-lang/vim-rescript'
+```
 
-" Packer
-use 'rescript-lang/vim-rescript'
+```lua
+-- Lazy.nvim
+{ 'rescript-lang/vim-rescript' }
```

Comment:
does it make sense to provide the `ft` option as well for lazy loading purposes? (disclaimer: I am not a lazy.vim specialist)

```suggestion
{ 'rescript-lang/vim-rescript', ft="rescript" }
```

Reply:
yup